python/mosaic/api/gdal.py

```diff
@@ -0,0 +1,59 @@
+from pyspark.sql import SparkSession
+from typing import Any
+
+__all__ = [
+    "setup_gdal",
+    "enable_gdal"
+]
+
+def setup_gdal(
```

Comment:
@edurdevic and @milos-colic After setup_gdal executes, recommend there is a print statement that lists where the init script was generated and a tip to add to cluster, restart, and then can call enable_gdal. Eventually, this can also include a mosaic doc link that explains in more detail.

Reply:
will add this. good point!